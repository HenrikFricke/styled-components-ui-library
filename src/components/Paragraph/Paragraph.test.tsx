import * as React from 'react';
import ReactTestRenderer from 'react-test-renderer';

import 'jest-styled-components';

import Paragrah from './Paragraph';

describe('Paragrah', () => {
    let tree: ReactTestRenderer.ReactTestRendererJSON;

    beforeEach(() => {
        tree = ReactTestRenderer.create(
            <Paragrah>I'am a small text</Paragrah>
        ).toJSON();
    });

    it('should render correctly', () => {
        expect(tree).toMatchStyledComponentsSnapshot();
    });
});