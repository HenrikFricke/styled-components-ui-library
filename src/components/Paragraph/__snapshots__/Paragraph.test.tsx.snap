// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragrah should render correctly 1`] = `
.fDNBJx {
  color: black;
  padding: 5px;
  font-size: 16px;
}

<p
  className="sc-bdVaJa fDNBJx"
>
  I'am a small text
</p>
`;
